src/common/options.cc

```diff
@@ -4657,11 +4657,15 @@ std::vector<Option> get_global_options() {
 
     Option("bluestore_avl_alloc_bf_threshold", Option::TYPE_UINT, Option::LEVEL_DEV)
     .set_default(131072)
-    .set_description(""),
+    .set_description("Sets threshold at which shrinking max free chunk size triggers enabling best-fit mode.")
+    .set_long_description("AVL allocator works in two modes: near-fit and best-fit. By default, it uses near-fit mode, in which it tries to fit a new block near last allocated block of similar size. This method is very fast. If memory gets fragmented AVL allocator switches to best-fit mode, in which it tries to find exact match for requested allocation. This method is considerably slower. When largest free block is smaller than 'bluestore_avl_alloc_bf_threshold', best-fit mode is used.")
+    .add_see_also("bluestore_avl_alloc_bf_free_pct"),
 
     Option("bluestore_avl_alloc_bf_free_pct", Option::TYPE_UINT, Option::LEVEL_DEV)
     .set_default(4)
-    .set_description(""),
+    .set_description("Sets threshold at which shrinking free space (in %, integer) triggers enabling best-fit mode.")
+    .set_long_description("AVL allocator works in two modes: near-fit and best-fit. By default, it tries to fit a new block near last allocated block of similar size. This method is very fast. If memory becomes scarce, it switches to best-fit mode, in which it tries to find exact match for requested allocation. This method is considerably slower. When free space is smaller than 'bluestore_avl_alloc_bf_free_pct', best-fit mode is used.")
```

Comment:
Same comment as above - wording here is just slightly different. I assume when either threshold is hit, best-fit mode is used. It might help to say that explicitly.